src/types-redirect.html

```diff
@@ -0,0 +1,41 @@
+<script>
```

Comment:
This is nice. I usually don't think about the disruption of moving all the information around. I'd rather this be done via an mdbook plugin, but since that plugin doesn't exist, this is fine. Speaking of which, we really need to update our books to mdbook 0.2.x.

Reply:
Is it correct that all the books will need to upgraded to 0.2 at the same time?  Otherwise, I imagine the rust-lang/rust "rustbook" tool would need to handle both versions, which sounds troublesome.

I'd be happy to help make that happen.  I feel like it would be best to make a tool to automatically update the books to make life easier.